packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -2345,6 +2345,19 @@
       }
     }
   },
+  "cloud-security-posture-settings": {
+    "dynamic": false,
+    "properties": {
+      "rules_states": {
+        "type": "object",
+        "properties": {
+          "muted": {
+            "type": "boolean"
+          }
+        }
+      }
+    }
```

Comment:
@kfirpeled, I didn't ignore the rule_id, since we maintain an array of objects, we are not going to search over the keys, therefore I didn't map them.

Reply:
we discussed that offline and agreed to remove the mappings for now